app/wrappers/DetectDataChange.js

```diff
@@ -47,6 +47,9 @@ class DetectDataChanges {
                 } else if (req.body.executorName) {
                     const currentExecutors = executorsWrapper.executors(true).map(executor => executor.fullName);
                     return this.isNotEqual(req.body.executorName, currentExecutors);
+                } else if (req.body.executorsApplying) {
+                    const executorsApplying = executorsWrapper.executorsApplying(true).map(executor => executor.fullName);
+                    return this.isNotEqual(req.body.executorsApplying, executorsApplying) && req.session.haveAllExecutorsDeclared !== 'true';
```

Comment:
I don't think we need to use ` && req.session.haveAllExecutorsDeclared !== 'true'` in the return statement here.